### PR TITLE
Give all old forms a created and updated date

### PR DESCRIPTION
**Pull request by 0atman**
Thanks to @thomasiles for suggesting this in #43

### CODE REVIEW

**Review by DavidBiddle — APPROVED** [review]
Looks good to me 🎉

**Comment by 0atman** [comment]
Do merge this before Monday if you would like the migration to happen. Otherwise I'll do so then!

File: spec/db/migration_7_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  include_context "with database"
+
+  let(:migrator) { Migrator.new }
+
+  before do
+    migrator.migrate_to(database, 6)
+  end
+  it "Update old forms with a created and updated date" do
+
+    @database[:forms].where(created_at: nil).update(
```
Comment (0atman) [inline spec/db/migration_7_spec.rb:11]: Wait don't merge, I just noticed I don't explicityly create any test cases. Though there SHOULD be stuff in the db to test, I'd want to add some.

**Comment by 0atman** [comment]
oh good my change failed rubocop, please don't fix!

**Review by aliuk2012 — CHANGES_REQUESTED** [review]
Am I missing something, where is the migration 7 file?

**Comment by 0atman** [comment]
It's dB test 7, not migration test 7. Maybe that naming is confusing...

On August 18, 2022, GitHub ***@***.***> wrote:
> @aliuk2012 requested changes on this pull request.
>
> Am I missing something, where is the migration 7 file?
>

—
Reply to this email directly, view it on GitHub
<https://github.com/alphagov/forms-api/pull/57#pullrequestreview-
1076887891>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAA33MNMOBGIFAR6YLATLFTVZXWI7ANCNFSM562I46JA>.
You are receiving this because you were assigned.Message ID:
***@***.***>

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+RUN apk --no-cache add postgresql-dev postgresql-libs postgresql-client less
+
+WORKDIR /app
+COPY .ruby-version Gemfile Gemfile.lock ./
+RUN apk --no-cache add alpine-sdk && bundle install && apk del alpine-sdk
+
+ADD . /app
+
+EXPOSE 9292
+CMD bundle exec rackup
```
Comment (aliuk2012) [inline Dockerfile:12]: I dont think this should of been part of your changes

File: Gemfile [hunk 3]
```diff
@@ -7,7 +7,7 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 gem "dotenv", "~> 2.7.6"
 gem "pg", "~> 1.3.5"
 gem "puma", "~> 5.0"
-gem "sentry-ruby", "~> 5.3.0"
+gem "sentry-ruby", "~> 5.4.0"
```
Comment (aliuk2012) [inline Gemfile:10]: I dont think this should of been part of your changes

File: manifest.yml [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
+    memory: 256M
+    command: bundle exec rackup -o 0.0.0.0 -p $PORT
+    services:
+      - forms-api-((PAAS_ENVIRONMENT))-db
+      - forms-api-((PAAS_ENVIRONMENT))-splunk
+    env:
+      RACK_ENV: production
+      SENTRY_DSN: ((SENTRY_DSN))
+      API_KEY: ((API_KEY))
+      PAAS_ENVIRONMENT: ((PAAS_ENVIRONMENT))
```
Comment (aliuk2012) [inline manifest.yml:13]: I dont think this should of been part of your changes

**Comment by 0atman** [comment]
sorry folks I'm not used to the new merge strategy on git yet, I pulled in some commits from main that I'm to lazy to remove. New PR incoming

**Comment by 0atman** [comment]
https://github.com/alphagov/forms-api/pull/60

**Pull request closed without merging**